source/simulator/stokes_matrix_free.cc

```diff
@@ -721,10 +721,18 @@ namespace aspect
 
         velocity.reinit (cell);
         velocity.read_dof_values (src.block(0));
+#if DEAL_II_VERSION_GTE(10,0,0)
+        velocity.evaluate (EvaluationFlags::gradients);
```

Comment:
Is this new in 10 or already in 9.3?

Reply:
The EvaluationFlags is in 9.3.
Change to DEAL_II_VERSION_GTE(9.3.0)?

Reply:
Yes, exactly. Because starting with 9.3 we can and should use the new method.

Reply:
Done!